cwms_radar_api/src/main/java/cwms/radar/data/dao/StoreRule.java

```diff
@@ -9,13 +9,14 @@
 	DO_NOT_REPLACE(CWMS_UTIL_PACKAGE.DO_NOT_REPLACE.toString()),
 	REPLACE_MISSING_VALUES_ONLY(CWMS_UTIL_PACKAGE.REPLACE_MISSING_VALUES_ONLY.toString()),
 	REPLACE_WITH_NON_MISSING(CWMS_UTIL_PACKAGE.REPLACE_WITH_NON_MISSING.toString()),
-	DELETE_INSERT(CWMS_UTIL_PACKAGE.DELETE_INSERT.toString());
-
+	DELETE_INSERT(CWMS_UTIL_PACKAGE.DELETE_INSERT.toString());	
 	private final String rule;
 
 	StoreRule(String rule)
 	{
-		this.rule = rule;
+		System.out.println(rule);
```

Comment:
Remove the System.out

Reply:
Thank you. You know the really sad part is that probably would've shown up nicely in the checkstyle report if we didn't still have a bunch of things to fix. (Thanks for slowly fixing those each PR btw.)